safety_limiter/src/safety_limiter.cpp

```diff
@@ -641,28 +692,31 @@ class SafetyLimiterNode
   }
   void cbCloud(const sensor_msgs::PointCloud2::ConstPtr& msg)
   {
-    pcl::PointCloud<pcl::PointXYZ>::Ptr pc(new pcl::PointCloud<pcl::PointXYZ>());
+    sensor_msgs::PointCloud2 cloud_msg_from_fixed;
 
     try
     {
-      sensor_msgs::PointCloud2 pc2_tmp;
-      geometry_msgs::TransformStamped trans = tfbuf_.lookupTransform(
-          frame_id_, msg->header.frame_id, msg->header.stamp, ros::Duration(0.1));
-      tf2::doTransform(*msg, pc2_tmp, trans);
-      pcl::fromROSMsg(pc2_tmp, *pc);
+      geometry_msgs::TransformStamped cloud_to_fixed = tfbuf_.lookupTransform(
+          fixed_frame_id_, msg->header.frame_id, msg->header.stamp, ros::Duration(0.1));
```

Comment:
This delay has larger impact than one in `predict()`.

Reply:
Exactly. The accepted timeout should be decided by the predict timer frequency `hz_`.
I modified the timeout on https://github.com/at-wat/neonavigation/pull/421/commits/137ff62c2a837d1e24a66abe7281aadb7a40bf8b

The reason why I set `ros::Duration(0.5 / hz_)` is:
- `ros::Duration(1.0 / hz_)` is too long because the process after this line in the `cbCloud()` will take.
- Removing `ros::Duration(1.0)` itself tends to lost the input cloud because it it too strict.